src/prop.rs

```diff
@@ -26,8 +26,7 @@ pub fn store_properties<
     modify_json(&file, |previous_data: &mut Option<S>| {
         match previous_data {
             Some(previous_data) => {
-                // HACK: Find how to handle case where simultaneous writing happens. What is the expected result
-                // Overwrite the data for now
+                // Overwrites data
```

Comment:
No no, in this place we shouldn't overwrite.

We can and should overwrite plain metadata, but in this place we work with properties, i.e. user data.

Reply:
@kirillt Ok for not overwriting, but what's the behavior you want